sealedenumprocessor/src/main/kotlin/com/livefront/sealedenum/internal/SealedClassExtensions.kt

```diff
@@ -0,0 +1,26 @@
+package com.livefront.sealedenum.internal
+
+import com.livefront.sealedenum.SealedEnum
+import com.squareup.kotlinpoet.ClassName
+
+typealias SealedClass = ClassName
+
+typealias SealedObject = ClassName
+
+/**
+ * Creates the name of the [SealedEnum] for the given [SealedClass] and [enumPrefix].
```

Comment:
Total optional but when looking back on stuff like this that I write I always find having one explicit example to be helpful. For example, here is something similar I've done myself:

```kotlin
        // Gets the class name without the package.
        // Ex: com.example.Parent.Child -> Parent.Child
        val packageName = type.rawType.`package`?.name ?: return null
        val classNameWithoutPackage = type
            .rawType
            .canonicalName!!
            .replace("$packageName.", "")

        // Gets the class name that would be generated by PolymorphicProcessor
        // Ex: Parent.Child -> Parent_Child_PolymorphicDataContainer
        val generatedClassSimpleName = classNameWithoutPackage.replace(".", "_")
        val generatedClassName =
            "$packageName.${generatedClassSimpleName}_PolymorphicDataContainer"
```

Just easier to glance at it to know exactly what it's doing without having to reverse engineer it your mind.

Reply:
Added examples in c56eeab!